.travis.yml

```diff
@@ -17,9 +17,6 @@ matrix:
 
   fast_finish: true
 
-  allow_failures:
-    - php: 7.1
```

Comment:
I think this never had proper tests, thus the allow_failures for the current  (7.2) then.

Reply:
but it doesn't fail on 7.2, so why would we allow_failures there?

Reply:
Yeah the problem is the single entry in PHP list :) Always needs min+max to be valid and running for those.